frontend/src/app/_directives/title.directive.ts

```diff
@@ -0,0 +1,29 @@
+import 'rxjs/add/operator/filter';
```

Comment:
I'll add some documentation. This directive looks for a page title set in the route and passes that to the template.